core/common/src/MapLayerSettings.ts

```diff
@@ -135,11 +130,41 @@ export interface MapLayerProps {
   transparentBackground?: boolean;
   /** Is a base layer.  Defaults to 'false'. */
   isBase?: boolean;
```

Comment:
Does `isBase` belong here, or on ImageMapLayerProps? Can a model serve as the base layer? BaseMapLayerSettings extends ImageMapLayerSettings, which suggests not.

Reply:
> Does `isBase` belong here, or on ImageMapLayerProps? Can a model serve as the base layer? BaseMapLayerSettings extends ImageMapLayerSettings, which suggests not.

Yes - this belongs here and is intentional.   A base layer must be complete as with Bing/Mapbox etc. or a color- it cannot contain completely transparent sections as in a model layer.

Reply:
On further thought, I misread your first comment --I'll fix this.

Reply:
isBase moved to ImageMapLayerProps.